src/components/RasterInfoPopup.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react";
```

Comment:
Maybe all this file contains too much duplicate code from file FeatureInfoPopup.js.
Have you attempted to use FeatureInfo popup forboth wms and rasters and decide in that component to draw rasters or wmslayer data (or maybe both at the same time)?

Reply:
Combined this file into the FeatureInfoPopup.js file.